src/serializers/mod.rs

```diff
@@ -148,21 +148,21 @@ impl SchemaSerializer {
     pub fn __repr__(&self) -> String {
         format!(
             "SchemaSerializer(serializer={:#?}, slots={:#?})",
-            self.serializer, self.slots
+            self.serializer, self.definitions
         )
     }
 
     fn __traverse__(&self, visit: PyVisit<'_>) -> Result<(), PyTraverseError> {
         self.serializer.py_gc_traverse(&visit)?;
-        for slot in self.slots.iter() {
+        for slot in self.definitions.iter() {
             slot.py_gc_traverse(&visit)?;
         }
         Ok(())
     }
 
     fn __clear__(&mut self) {
         self.serializer.py_gc_clear();
-        for slot in self.slots.iter_mut() {
+        for slot in self.definitions.iter_mut() {
             slot.py_gc_clear();
```

Comment:
```suggestion
        for definition in self.definitions.iter_mut() {
            definition.py_gc_clear();
```

Reply:
But definitions doesn’t have that method right? It could I suppose. Truth be told when I first implemented this GC stuff I had the idea that we would eventually split it out into its own trait but never got around to it

Reply:
Oh never mind, I see now